src/core/agent/services/multiSigService.ts

```diff
@@ -615,6 +617,46 @@ class MultiSigService extends AgentService {
     };
   }
 
+  async membersReadyToRotate(identifierId: string): Promise<string[]> {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifierId
+    );
+
+    const multiSig = await this.props.signifyClient
+      .identifiers()
+      .get(metadata.signifyName);
+
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(metadata?.signifyName);
+
+    const nextSequence = (Number(multiSig.state.s) + 1).toString();
+
+    const rmids = members?.rotation;
+
+    const rstates: any[] = [];
+    await Promise.allSettled(
+      rmids.map(async (rotation: any) => {
+        const op = await this.props.signifyClient
+          .keyStates()
+          .query(rotation.aid, nextSequence);
+        await waitAndGetDoneOp(this.props.signifyClient, op);
+        if (op.done) {
+          rstates.push(op.response);
+        }
+      })
+    );
+
+    const rotated: string[] = [];
+    for (const rstate of rstates) {
+      if (rstate.k.join(",") !== multiSig.state.k.join(",")) {
```

Comment:
Why would these ever be equal? `rstate` is for a single sig identifier. `multiSig` has all of the rstates combined. Should it not be `multiSig.state.k.includes(state.k[0])`?

Reply:
^ please test this out fully